watertap/unit_models/zero_order/__init__.py

```diff
@@ -11,6 +11,7 @@
 #
 ###############################################################################
 
+from .autothermal_hydrothermal_liquefaction_zo import ATHTLZO
```

Comment:
Minor detail, but I have been putting all zero order units for AMO at the bottom section, separately from the ones derived from WT3.

Reply:
Yeah, I saw the AMO category and moved these new models to it.